tests/test-scribe.c

```diff
@@ -359,8 +359,9 @@ assert_first_mib_zeroed (Fixture *fixture)
   g_assert_no_error (error);
   g_assert (ret);
 
+  g_assert_cmpuint (target_length, >=, ONE_MIB);
```

Comment:
`<=`, no?

Reply:
No, the file on disk or memfd is always larger than 1 MiB.

Reply:
Pushed one more patch.

Reply:
Hmm, I was going by the `g_input_stream_read_all()` call in the memfd case, where you definitely wouldn’t expect `target_length > ONE_MIB`, since that would mean it had read off the end of the buffer it was provided. But asserting `target_length >= ONE_MIB` does work for both cases if you ignore the possibility of reading off the end of the buffer, and want to catch `target_length < ONE_MIB` as an error (which makes sense).

👍

Reply:
I guess one could make this clearer by floating the two slightly different assertions into each block of the test. I'll take the liberty of doing that then squashing & merging.